static/app/views/settings/project/server-side-sampling/samplingFromOtherProject.spec.tsx

```diff
@@ -0,0 +1,63 @@
+import {act, render, screen} from 'sentry-test/reactTestingLibrary';
+
+import ProjectsStore from 'sentry/stores/projectsStore';
+import {ServerSideSamplingStore} from 'sentry/stores/serverSideSamplingStore';
+
+import {SamplingFromOtherProject} from './samplingFromOtherProject';
+import {getMockData, mockedSamplingDistribution} from './testUtils';
+
+export const samplingBreakdownTitle = 'Transaction Breakdown';
+
+describe('Server-Side Sampling - SamplingFromOtherProject', function () {
+  afterEach(function () {
+    act(() => ProjectsStore.reset());
```

Comment:
do we need `act` here?

Reply:
it was failing without it and I've seen other tests fighting that with an act

Reply:
if you use 

` beforeEach(function () {
    ProjectsStore.reset();
    ServerSideSamplingStore.reset();
  });
` 

it seems to solve the issue. ANother alternative would be to have this reset functions inside of each test